programs/just.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, pkgs, config, ... }:
+let
+  cfg = config.programs.just;
+in
+{
+  options.programs.just = {
+    enable = lib.mkEnableOption "just";
+    package = lib.mkPackageOption pkgs "just" { };
+  };
+
+  config = lib.mkIf cfg.enable {
+    settings.formatter.just = {
+      command = cfg.package;
+      options = [
+        "--fmt"
+        "--unstable" # formtting is still unstable
+        "--justfile" # takes the file as argument, treefmt will append it
```

Comment:
Is it able to take multiple files at once? This is require to respect the treefmt formatter spec.

Reply:
No, it is not. :(